docs/data/data-grid/row-updates/InfiniteLoadingGrid.tsx

```diff
@@ -31,7 +32,7 @@ export default function InfiniteLoadingGrid() {
     setLoading(true);
     const newData = await getRealGridData(newRowLength, getCommodityColumns());
     // Simulate network throttle
-    await sleep(Math.random() * 500 + 100);
+    await sleep(randomInt(100, 600));
```

Comment:
`randomInt` would generate the same value when running in the test environment:

https://github.com/mui/mui-x/blob/2d82a6f43b504799c8ea026e5a41d0717d7cbe49/test/regressions/webpack.config.js#L25